tekton/release-cheat-sheet.md

```diff
@@ -19,6 +19,8 @@ the pipelines repo, a terminal window and a text editor.
     ```bash
```

Comment:
Can we also remove the previous sentence `Ensure the correct version of the release pipeline is installed on the cluster` ?